tests/cython/test_cython.pyx

```diff
@@ -0,0 +1,20 @@
+# cython: language_level=3
+import trio
+
+async def foo() -> None:
+    print('.')
+
+async def trio_main() -> None:
+    print('hello...')
+    await trio.sleep(1)
+    print(' world !')
+
+    async with trio.open_nursery() as nursery:
+        nursery.start_soon(foo)
+        nursery.start_soon(foo)
+        nursery.start_soon(foo)
+
+def main() -> None:
+    trio.run(trio_main)
+
+main()
```

Comment:
```suggestion
if __name__ == "__main__":
    main()
```

maybe? not sure why there's a `main` function to begin with

Reply:
lol, good question.